pkg/cmd/cli/backup/create_test.go

```diff
@@ -127,3 +133,153 @@ func TestCreateOptions_OrderedResources(t *testing.T) {
+	flags.Parse([]string{"--wait"})
+
+	args := []string{name, "arg2"}
+	o.Complete(args, f)
+	e := o.Validate(cmd, args, f)
+	assert.Contains(t, e.Error(), "/api?timeout=")
+
+	e = o.Run(cmd, f)
+	// as fromSchedule is set, so the error below is expected
+	assert.Contains(t, e.Error(), fmt.Sprintf("Get \"%s/apis/velero.io/v1/namespaces/%s/schedules/%s", clicmd.HOST, clicmd.VeleroNameSpace, fromSchedule))
+
+	// verify all options are set as expected
+	assert.Equal(t, name, o.Name)
+	assert.Equal(t, includeNamespaces, o.IncludeNamespaces.String())
+	assert.Equal(t, excludeNamespaces, o.ExcludeNamespaces.String())
+	assert.Equal(t, includeResources, o.IncludeResources.String())
+	assert.Equal(t, excludeResources, o.ExcludeResources.String())
+	assert.Equal(t, includeClusterScopedResources, o.IncludeClusterScopedResources.String())
+	assert.Equal(t, excludeClusterScopedResources, o.ExcludeClusterScopedResources.String())
+	assert.Equal(t, includeNamespaceScopedResources, o.IncludeNamespaceScopedResources.String())
+	assert.Equal(t, excludeNamespaceScopedResources, o.ExcludeNamespaceScopedResources.String())
+	assert.Equal(t, true, clicmd.CompareSlice(strings.Split(labels, ","), strings.Split(o.Labels.String(), ",")))
+	assert.Equal(t, storageLocation, o.StorageLocation)
+	assert.Equal(t, snapshotLocations, strings.Split(o.SnapshotLocations[0], ",")[0])
+	assert.Equal(t, selector, o.Selector.String())
+	assert.Equal(t, orderedResources, o.OrderedResources)
+	assert.Equal(t, csiSnapshotTimeout, o.CSISnapshotTimeout.String())
+	assert.Equal(t, itemOperationTimeout, o.ItemOperationTimeout.String())
+	assert.Equal(t, snapshotVolumes, o.SnapshotVolumes.String())
+	assert.Equal(t, snapshotMoveData, o.SnapshotMoveData.String())
+	assert.Equal(t, includeClusterResources, o.IncludeClusterResources.String())
+	assert.Equal(t, defaultVolumesToFsBackup, o.DefaultVolumesToFsBackup.String())
+	assert.Equal(t, resPoliciesConfigmap, o.ResPoliciesConfigmap)
+	assert.Equal(t, dataMover, o.DataMover)
+	assert.Equal(t, fromSchedule, o.FromSchedule)
+	assert.Equal(t, true, o.Wait)
+
+	// verify oldAndNewFilterParametersUsedTogether
+	mix := o.oldAndNewFilterParametersUsedTogether()
+	assert.Equal(t, true, mix)
+
+	// create the other create command without fromSchedule option for Run() other branches
+	cmd = NewCreateCommand(f, "velero backup create")
+	assert.Equal(t, "Create a backup", cmd.Short)
+
+	o = NewCreateOptions()
+	o.Labels.Set("velero.io/test=true")
+	o.OrderedResources = "pods=p1,p2;persistentvolumeclaims=pvc1,pvc2"
+	o.CSISnapshotTimeout = 20 * time.Minute
+	o.ItemOperationTimeout = 20 * time.Minute
+	o.Wait = true
+	f.BindFlags(cliFlags)
+	args = []string{"backup-name-2", "arg2"}
+	e = o.Complete(args, f)
+	assert.NoError(t, e)
+	fmt.Println(o.client)
+	e = o.Run(cmd, f)
+
+	// Get failure of backup resource creation
+	assert.Contains(t, e.Error(), fmt.Sprintf("Post \"%s/apis/velero.io/v1/namespaces/%s/backups", clicmd.HOST, clicmd.VeleroNameSpace))
```

Comment:
Thanks @danfengliu for the effort that tries to improve the code coverage.
But we should mock the dependencies and cover the code as much as possible rather than just checking the error.

If that is impossible or not easy, I suggest we skip the UT for such code at this moment.